environment.yml

```diff
@@ -12,4 +12,5 @@ dependencies:
   - scipy=1.*
   - tqdm
   - xarray
+  - dask
```

Comment:
Should we maybe make it an optional dependency? I don't know how much more dependencies dask involves...

Reply:
Looks like it's pretty much pandas/numpy, which we already have. Could install only `dask-core` to ensure it's minimal.

Reply:
Keeping `dask` for now.